packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -17,19 +17,19 @@ import { MAINAREA_TABBAR_CONTEXT_MENU } from './shell';
 
 export namespace CommonMenus {
 
-    export const EDIT_MENU = "2_edit";
-    export const EDIT_MENU_UNDO_GROUP = "1_undo/redo";
-    export const EDIT_MENU_CUT_COPY_PASTE_GROUP = "2_cut/copy/paste";
-    export const EDIT_MENU_FIND_REPLACE_GROUP = "3_find/replace";
+    export const FILE = [MAIN_MENU_BAR, '1_file'];
+    export const FILE_NEW = [...FILE, '1_new'];
```

Comment:
`new` and `open ...` are very specific to editor and workspace. Shouldn't the declaration stay there?

Reply:
ok

Reply:
@svenefftinge With open is easy, but new is used into the terminal extension. Could we move `new terminal` to `view` menu?

Reply:
+1 Just like in VS Code.